### PR TITLE
Enable head tests passing by default

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Enable head tests passing by default

### CODE REVIEW

File: test/commands/solution/search_community_solutions_test.rb [hunk 1]
```diff
@@ -161,9 +161,12 @@ class Solution::SearchCommunitySolutionsTest < ActiveSupport::TestCase
...
     exercise = create :concept_exercise, track: track
-    solution_1 = create :concept_solution, :published, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash
-    solution_2 = create :concept_solution, :published, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash
-    solution_3 = create :concept_solution, :published, exercise: exercise, git_important_files_hash: 'different_hash'
+    solution_1 = create :concept_solution, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash,
+num_stars: 11, published_at: Time.current, status: :published
+    solution_2 = create :concept_solution, exercise: exercise, git_important_files_hash: exercise.git_important_files_hash,
+num_stars: 22, published_at: Time.current, status: :published
+    solution_3 = create :concept_solution, exercise: exercise, git_important_files_hash: 'different_hash', num_stars: 33,
+published_at: Time.current, status: :published
```
Comment (iHiD) [inline test/commands/solution/search_community_solutions_test.rb]: Can you indent these pls.

Comment (ErikSchierboom) [reply test/commands/solution/search_community_solutions_test.rb]: I did. Rubocop changed them back. I'll force it

**Comment by iHiD** [comment]
Are there any statuses we need to check are correct everywhere before merging this?

**Comment by iHiD** [comment]
If the git important files has changes, BUT the latest tests pass, then auto-update the solution. And have a preference setting for this.

**Comment by ErikSchierboom** [comment]
> And have a preference setting for this.

Should we create a new settings tab/sub-page for this? And if so: what do we name it?

**Comment by iHiD** [comment]
Add a `Preferences` tab.

File: app/assemblers/assemble_exercise_community_solutions_list.rb [hunk 2]
```diff
@@ -26,7 +26,7 @@ def solutions
       criteria: params[:criteria],
       sync_status: params[:up_to_date].present? ? :up_to_date : nil,
       tests_status: params[:passed_tests].present? ? :passed : nil,
-      head_tests_status: params[:passed_head_tests].present? ? %i[not_queued queued passed] : nil
+      head_tests_status: params[:not_passed_head_tests].present? ? nil : %i[queued passed]
```
Comment (ErikSchierboom) [inline app/assemblers/assemble_exercise_community_solutions_list.rb]:
```suggestion
      head_tests_status: params[:not_passed_head_tests].present? ? nil : %i[not_queued queued passed]
```

**Review by iHiD — APPROVED** [review]

**Pull request merged**